buildroot/share/PlatformIO/scripts/common-dependencies.py

```diff
@@ -322,6 +322,128 @@ def check_configfile_locations():
 				err = 'ERROR: Config files found in directory ' + str(p) + '. Please move them into the Marlin subdirectory.'
 				raise SystemExit(err)
 
+# Get the board being built from the Configuration.h file
+#   return: board name
+def get_board_name():
```

Comment:
All of this can be replaced by using parsed  `env['MARLIN_FEATURES']['MOTHERBOARD']`

Reply:
I looked but didn't see that...    using.